### DISCUSSION TITLE
What is Debug.enableRefactor?

### DISCUSSION THREAD

**Discussion by aperullo** [community/community#6853] [Copilot Conversations]
One of the options when editing `settings.json` is 

```
"github.copilot.advanced": {
  "debug.enableRefactor": true/false
}
```

The description is "enable refactor status bar item for Python". I enabled this setting, restarted and opened a python file, but my status bar looks unchanged.

What does this setting do and how do I enable it to play with?

**Comment by theodore-evans** [comment]
Could this maybe be a placeholder for modify support, like the feature demonstrated on the openai da-vinci-codex beta playground?
